ingest/a_vs_b_de.py

```diff
@@ -0,0 +1,65 @@
+#using https://nbviewer.org/github/theislab/diffxpy_tutorials/blob/master/diffxpy_tutorials/test/multiple_tests_per_gene.ipynb
+import anndata
+import matplotlib.pyplot as plt
+import seaborn as sns
+import logging
+import numpy as np
+import pandas as pd
+import scipy.stats
+import scanpy as sc
+import os
+
+
+#install with pip install batchglm -- NOT using github repo
+import batchglm.api as glm
+
+#install with pip install diffxpy -- NOT using github repo
+import diffxpy.api as de
+
+#check that batchglm and diffpy were installed correctly
+#print("batchglm version "+glm.__version__)
+#print("diffpy version "+de.__version__)
+
+#Pairwise tests between groups
+#answers whether a given pair of groups shows differential expression for each gene
+
+#arguments:
+#data: Anndata object, data matrix with cells x genes
+
+#grouping: str, column in data, the column that contains cell type labels. alternatively vector containing group labels
+
+#test: type of test, default is z-test. other options: ’wald’, ’lrt’, ’t-test’, ’rank’
+
+#lazy: bool, only possible if test is ztest, if true only evaluated once the user requests pval/coefficients for a specific pair of models
+
+#noise_model: default is nb, specify NONE for wald and t test
+
+h5ad_file = "/Users/mvelyuns/scp-ingest-pipeline/tests/data/anndata/trimmed_compliant_pbmc3K.h5ad"
+
+adata_file = sc.read_h5ad(h5ad_file)
+
+adata_file.var_names_make_unique()
+
+#print(adata_file.obs["louvain"])
+
+test = de.test.pairwise(
+    data= adata_file,
+    grouping="louvain",
+    test="t-test",
```

Comment:
For "A vs all" [we use ScanPy](https://singlecell.zendesk.com/hc/en-us/articles/6059411840027-Exploratory-differential-gene-expression-analysis) with the [Wilcoxon Rank-Sum (Mann-Whitney U Test)](https://en.wikipedia.org/wiki/Mann%E2%80%93Whitney_U_test), a non-parametric test. The "wilcoxon" option for the pairwise comparisons is called ["rank"](https://diffxpy.readthedocs.io/en/latest/api/diffxpy.api.test.pairwise.html). For consistency, we should start with "A vs. B" using "rank" instead of "t-test". 

Blocker: Please adjust this PR to use "rank" and confirm that the syntax here is sufficient for the wilcoxon test. (We saw while pairing that the "wald" test requires a string to specify the generalized linear model it should be testing)

Reply:
changing test from "t-test" to "rank" works now!